src/common/adapters/pivotal.js

```diff
@@ -1,55 +0,0 @@
-import {
-  $all,
-  $classed,
```

Comment:
Looks like we could now even throw out some functions from `src/common/adapters/helpers.js` like `classed` and `closest` 🗑

Reply:
Neat! I checked which ones were not used and removed them ✅